libs/navigation/src/facades/navigation.facade.ts

```diff
@@ -25,10 +25,11 @@ export class DaffNavigationFacade implements DaffNavigationFacadeInterface {
    */
   errors$: Observable<string[]>;
 
-  constructor(private store: Store<NavigationReducersState>) {
-    this.tree$ = this.store.pipe(select(selectNavigationTree));
-    this.loading$ = this.store.pipe(select(selectNavigationLoading));
-    this.errors$ = this.store.pipe(select(selectNavigationErrors));
+  constructor(private store: Store<DaffNavigationReducersState<T>>) {
+		const selectors = getDaffNavigationSelectors<T>();
```

Comment:
For consistency with how NgrxEntity works, I think we should import like:

```ts
const { selectNavigationTree, selectNavigationLoading, selectNavigationErrors } = getDaffNavigationSelectors<T>();
```

Reply:
done